src/clj/fluree/db/query/exec/eval.cljc

```diff
@@ -25,192 +25,196 @@
 
 (defn sum
   [coll]
-  (reduce + coll))
+  (where/->typed-val (reduce + (mapv :value coll))))
```

Comment:
For `sum`, `avg`, `median`, etc where we are performing math, I think we should have a try*/catch* in case there is a non-numeric datatype, and output a friendly ex-message of that cause.

It doesn't need to be in here as it wasn't in here before these changes, but just noticing now.